frontend/src/app/shared/components/autocompleter/project-autocompleter/project-autocompleter.component.ts

```diff
@@ -140,9 +142,12 @@ export class ProjectAutocompleterComponent extends OpAutocompleterComponent<IPro
     const arrayedValue = (Array.isArray(normalizedValue) ? normalizedValue : [normalizedValue]).map((p) => p.href || p.id);
     return projects.map((project) => {
       const isSelected = !!arrayedValue.find((selected) => selected === this.projectTracker(project));
+      const id = project.id.toString();
+      const disabled = isSelected || project.disabled || !!this.disabledProjects[id];
```

Comment:
Hi, `!!this.disabledProjects[id]` counts entries with an empty string as not disabled e.g. `{ someProject: ""  }`

if we want to support empty strings we could do

```suggestion
      const disabled = isSelected || project.disabled || (id in this.disabledProjects && this.disabledProjects[id] !== false);
```

Reply:
From Rails, you could also pass a boolean instead of an empty string, so I'd be okay with treating "" as falsey. The argument behind it would be:

- If you want reasons, use strings only for those that you want shown
- If you want to disable without reason, use a boolean
- you can freely mix between the two

Reply:
It makes sense.